src/components/TakeAction.vue

```diff
@@ -78,17 +78,17 @@
         </v-expansion-panel-header>
 
         <v-expansion-panel-content>
-          <sign-name />
+          <sign-name @form-filled="handleFormFilled" />
         </v-expansion-panel-content>
 
         <v-expansion-panel-content>
           <v-btn
             width="160"
-            dark
             color="primary"
+            :disabled="!formFilled"
             @click="nextPage({ userData })"
           >
-            Next
+            {{ formFilled ? 'Next' : 'Please Fill Form!' }}
```

Comment:
I think we could just disable the button and not change the text. Perhaps along with adding a disabled cursor effect. 

https://www.w3schools.com/cssref/pr_class_cursor.php

Reply:
Thank you, I will do just that.